profile_library/innr/FL 122 C/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-10-11T10:41:49.998998",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly PlugS",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.14.10:docker"
+  },
+  "name": "FL 122 C",
```

```suggestion
  "name": "Innr Light Strip Colour",
```

This may be the marketed name. It's not used for model matching logic, but only in library information. https://library.powercalc.nl